5_1.py

```diff
@@ -0,0 +1,16 @@
+text = input("Write you text here: ")
+for
+if text.isupper():
+    print(f"you typed upper case letters {len(text)}")
+if text.islower():
+    print(f"you typed lower case letters {len(text)}")
+if text.isdigit():
+    print(f"you typed digits {len(text)}")
+    a = int(text)
+    if a % 2 == 0:
+        print ("Number id even")
+    else:
+        print ("number is odd")
+
+print (f"you typed {len(text)} char")
```

Comment:
here just else could be used
```suggestion
    # if not text.isdigit(), text is strig, or you can check it using .isalpha()
    else:
        print (f"you typed {len(text)} char")
```